src/main/java/com/microsoft/cse/helium/models/WebfluxConfig.java

```diff
@@ -0,0 +1,19 @@
+package com.microsoft.cse.helium.models;
+
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.reactive.config.ResourceHandlerRegistry;
+import org.springframework.web.reactive.config.WebFluxConfigurer;
+
+@Configuration
+public class WebfluxConfig implements WebFluxConfigurer {
+
+    @Override
+    public void addResourceHandlers(ResourceHandlerRegistry registry) {
+
+        registry.addResourceHandler("/swagger-ui.html**")
```

Comment:
Can we make / display the swagger UI like C# and Node do? Not a hard requirement, but a nice to have.

Reply:
The "/" is a bit broken in Webflux and the redirect did not work as swagger support is still not top notch with webflux. I looked for a while but will look for a bit longer. I did add that note to the PR saying this is broken and will be looked again.

Reply:
Also added a bug to the triage to track the same.